src/win32/pthread/pthread_mutex_destroy.c

```diff
@@ -37,7 +37,9 @@
 #include "pthread.h"
 #include "implement.h"
 
-int getpid(void);
+#if REALM_UWP
+int _getpid(void);
+#endif
```

Comment:
Is this file being compiled for anything but UWP now? If so, does this not change the old behaviour?